clang/lib/Parse/ParseStmt.cpp

```diff
@@ -832,10 +832,13 @@ StmtResult Parser::ParseCaseStatement(ParsedStmtContext StmtCtx,
           << "'case'" << tok::colon
           << FixItHint::CreateReplacement(ColonLoc, ":");
     } else {
-      SourceLocation ExpectedLoc = PP.getLocForEndOfToken(PrevTokLocation);
+      SourceLocation ExpectedLoc = getEndOfPreviousToken();
+
       Diag(ExpectedLoc, diag::err_expected_after)
           << "'case'" << tok::colon
-          << FixItHint::CreateInsertion(ExpectedLoc, ":");
+          << FixItHint::CreateInsertion(ExpectedLoc,
+                                        tok::getTokenName(tok::colon));
```

Comment:
Whoops, looks like this caused the word `colon` to be displayed instead of `:`.
https://godbolt.org/z/rjzdsKqqh

Reply:
@rkirsling I’ll check and fit it today. Thanks

Reply:
Sure! Checking locally, it looks like this line could be safely reverted. Lemme know if you'd prefer me to submit the fix instead.

Reply:
Opened #144069, since it's such a quick fix.